frame/treasury/src/lib.rs

```diff
@@ -323,11 +326,11 @@ decl_module! {
 		/// proposal is awarded.
 		///
 		/// # <weight>
-		/// - O(1).
-		/// - Limited storage reads.
-		/// - One DB change, one extra DB entry.
+		/// - Complexity: O(1)
+		/// - DbReads: `ProposalCount`, `sender account`
+		/// - DbWrites: `ProposalCount`, `Proposals`, `sender account`
 		/// # </weight>
-		#[weight = 500_000_000]
+		#[weight = 114_700_000 + T::DbWeight::get().reads_writes(1, 2)]
```

Comment:
so can you bring me up to speed, where are these numbers coming from? the `114_700_000 ` per se.

Reply:
- to me it seems like we ignore `sender account` all the time here. Correct? Also best to call it `origin` in the doc as well.

Reply:
this number directly comes from the benchmark server at https://www.shawntabrizi.com/substrate-graph-benchmarks/?p=treasury&e=tip_new